bedrock/firefox/tests/test_firefox_details.py

```diff
@@ -116,7 +114,7 @@ def setUp(self):
     def test_get_download_url(self):
         url = firefox_desktop.get_download_url('release', '17.0.1', 'osx', 'pt-BR', True)
```

Comment:
It seems the `version` argument is now only used for the `esr_next` channel. Is there a benefit to keeping the `version` argument in tests for other channels?

Reply:
It's just the way it's called from the `_get_filtered_builds` method. It's potentially possible to remove it, but I'd rather do a more thorough cleanup when/if we can also use an alias for `esr_next`.